package.json

```diff
@@ -50,9 +59,11 @@
     "@vitejs/plugin-react": "^4.0.4",
     "eslint": "8.49.0",
     "eslint-config-next": "13.5.1",
+    "graphql-tag": "^2.12.6",
     "happy-dom": "^12.1.5",
     "prettier": "^3.0.3",
     "rimraf": "^5.0.1",
+    "vite": "^4.4.9",
```

Comment:
What's `vite` used for?

Reply:
It was a warning for a peer dependency of something I can't remember right now.